code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -179,6 +179,21 @@
 		visible_message("<span class='notice'>[src] fails to expand!</span>")
 	qdel(src)
 
+/obj/item/reagent_containers/food/snacks/monkeycube/suicide_act(mob/user)
+	var/mob/living/M = user
+	user.visible_message("<span class='suicide'>[M] is putting [src] in [M.p_their()] mouth! It looks like [M.p_theyre()] trying to commit suicide!</span>")
+	sleep(10)
+	playsound(M.loc,'sound/items/eatfood.ogg', rand(10,50), TRUE)
+	if(HAS_TRAIT(M, TRAIT_NOHUNGER))
+		to_chat(user, "<span class='warning'>Your body won't activate [src]...</span>")
+		return SHAME
+	sleep(10)
+	Expand()
+	M.visible_message("<span class='danger'>[M]'s torso bursts open as a primate emerges!</span>")
+	ADD_TRAIT(M, TRAIT_DISFIGURED, TRAIT_GENERIC)
+	M.gib(null, TRUE, null, TRUE)
```

Comment:
why add disfigured if we just gib them

Reply:
most of my code is cobbled together from other, similar places, to prevent me missing anything, so i haven't the faintest idea